plugins/node/opentelemetry-instrumentation-ioredis/src/instrumentation.ts

```diff
@@ -96,4 +97,103 @@ export class IORedisInstrumentation extends InstrumentationBase<
       return traceConnection(this.tracer, original);
     };
   }
+
+  private defaultDbStatementSerializer: DbStatementSerializer = (
+    cmdName,
+    cmdArgs
+  ) =>
+    Array.isArray(cmdArgs) && cmdArgs.length
+      ? `${cmdName} ${cmdArgs.join(' ')}`
+      : cmdName;
+
+  private traceSendCommand = (
```

Comment:
another small suggestion: since you copied the `traceSendCommand` from utils to instrumentation, I think it can be nice to also copy the `traceConnection` function, so the codebase is consistent